src/SConscript

```diff
@@ -1378,6 +1378,7 @@ if 'opt' in needed_envs:
             LINKFLAGS = Split(ldflags['opt']),
             disable_partial=disable_partial)
 
+
```

Comment:
Revert this change.

Reply:
Sorry I'm a bit confused. Revert what?

Reply:
There's an extra newline, according to the diff here.